hr_job_category/README.rst

```diff
@@ -0,0 +1,103 @@
+==========================
+HR Job Employee Categories
+==========================
+
+.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Beta-yellow.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fhr-lightgray.png?logo=github
+    :target: https://github.com/OCA/hr/tree/11.0/hr_job_category
+    :alt: OCA/hr
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-11-0/hr-11-0-hr_job_category
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/11.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module was written to extend the functionality of jobs
+to support tagging employees based on their job positions.
+For example, all Supervisors could be attached to the Supervisors category.
+Define which categories a job belongs to in the configuration for the job.
+When an employee is assigned a particular job the categories attached to that
+job will be attached to the employee record as well.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Installation
+============
+
+To install this module, you need to:
+
+* clone the branch 8.0 of the repository https://github.com/OCA/hr
```

Comment:
Outdated instructions, but this shouldn't even be there as they are generic instructions.

Reply:
@pedrobaeza I don't see it in the fragment.
Although, there's no reason to explain the cloning etc.

Reply:
Then don't touch yourself the file, but generate it with the OCA/maintainer-tools script.